federatedscope/core/trainers/trainer_nbafl.py

```diff
@@ -54,8 +54,10 @@ def init_nbafl_ctx(base_trainer):
     cfg = base_trainer.cfg
 
     # set proximal regularizer
+    cfg.regularizer.defrost()
     cfg.regularizer.type = 'proximal_regularizer'
     cfg.regularizer.mu = cfg.nbafl.mu
+    cfg.regularizer.freeze(save=False)
```

Comment:
when should we set `save` to be `False`?

Reply:
In my opinion, the target of saving configuration is to reproduct the experiment. So maybe we should save the `config`  before calling `FedRunner().run()`

Reply:
But the config might be changed to "valid" value in some assertions and saved. I think the saved config implies what takes effect during the exp.

Reply:
> 

I have checked the code, and delete the `save` parameter in `cfg.freeze()`.